### DISCUSSION TITLE
Dynamic Trees (Plus and Quark) with Create mod

### DISCUSSION THREAD

**Discussion by SlenAJ** [EnigmaticaModpacks/CreateTogether#37] [Q&A] [answered]
Hi All,

My friends and I like using the Dynamic Trees mod (with Plus and Quark addons), which raises a question.  Does anyone know how Create's blocks, like the saw, will interact with the different stages of tree growth in Dynamic Trees?  I plan to test this out soon but wanted to put some feelers out also.

Any thoughts or assistance is much appreciated.  I will also report back here with any findings.

For reference, here are links to the CurseForge pages for the mentioned mods:
  https://www.curseforge.com/minecraft/mc-mods/dynamictrees
  https://www.curseforge.com/minecraft/mc-mods/dynamictreesplus
  https://www.curseforge.com/minecraft/mc-mods/dynamic-trees-quark

-Slen

**Comment by NielsPilgaard** [comment, marked as answer]
I'm afraid I haven't tested it, but I imagine it'll work just fine.

**Reply by SlenAJ** [reply]
You are correct, it does work just fine.  My findings were that the Dynamic Trees mods adds an additional resource (seed).  This seed is what grows the Dynamic Trees type of trees.  By default, saplings are still in the game and they grow normal Minecraft trees.  
Create interacts with fully grown Dynamic Trees just fine also.
